src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/hotshard/HotShardClusterRca.java

```diff
@@ -138,7 +140,7 @@ private double getThresholdValue(Map<NodeShardKey, Double> perIndexShardInfo, do
      *
      */
     private void findHotShardAndCreateSummary(Table<String, NodeShardKey, Double> resourceInfoTable, double thresholdInPercentage,
-                                              List<HotResourceSummary> hotResourceSummaryList, ResourceType resourceType) {
+                                              List<HotResourceSummary> hotResourceSummaryList, Resource resourceType) {
```

Comment:
let's also rename the `resourceType` here.

Reply:
done